node-src/lib/utils.ts

```diff
@@ -3,19 +3,19 @@ import picomatch, { Matcher } from 'picomatch';
 export const lcfirst = (str: string) => `${str.charAt(0).toLowerCase()}${str.slice(1)}`;
 
 export const delay = (ms: number) => new Promise<void>((resolve) => setTimeout(resolve, ms));
-export const tick = async (times: number, interval: number, fn: (i: number) => any) => {
-  for (let i = 0; i < times; i += 1) {
+export const tick = async (times: number, interval: number, function_: (index: number) => any) => {
```

Comment:
Could be more descriptive than `function_` I think, maybe `callback`?

Reply:
Ope! Good find, not sure how I missed that one.